### DISCUSSION TITLE
Support for Division Information?

### DISCUSSION THREAD

**Discussion by joeyagreco** [mattdodge/yahoofantasy#54] [General]
I'm using this for Fantasy Football.

Was wondering if there is anywhere to retrieve division information for a league?

Don't see it anywhere in the [league](https://github.com/mattdodge/yahoofantasy/blob/master/yahoofantasy/resources/league.py) model.

Any insight would be appreciated!

Thanks

**Comment by mattdodge** [comment]
Good question - I assume some of that comes back in the API but I'm not enrolled in a league with divisions so it's hard to confirm for sure. You can poke around the raw API response using the `_raw` attribute on whatever object you're curious about, see if there's anything good in there for division information

```python
league = ctx.get_leagues('nfl', 2022)[0]
print(league._raw)
```
